### PR TITLE
Fix #602: Refactor Train to Practice

### LINKED ISSUE #602: Train should be updated to Practice
As per new mocks, Topic Train Tab has been updated to Topic Practice Tab,

Screenshot 2020-01-03 at 12 11 29 PM

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Train` with `Practice`.

### PR DESCRIPTION
**Pull request by Luffy18346**
## Explanation
Fix #602: Refactored Train to Practice in all related files. 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt]: Update this comment to `/** Fragment that displays skills for topic practice mode. */`

File: app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt
Comment (rt4914) [inline app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt]: Update this to:
`TODO(#159): Use this skillList from TopicPracticeFragment to fetch questions and start practice mode.`

File: app/src/main/res/layout/topic_practice_skill_view.xml
Comment (Luffy18346) [inline app/src/main/res/layout/topic_practice_skill_view.xml:20]: @rt4914,The line `    app:constraint_referenced_ids="button_example,text_view_status"` is deleted because the line was giving error as the ids mentioned in the line do not exist anymore in the file. App was running successfully without removing the line. But because of error, I removed it.

**Comment by Luffy18346** [comment]
@rt4914,I have made changes in Testing file `TopicTrainFragmentTest ` but they are failing on checking. Because  I haven't been able to degrade my android studio. Other than that, I have checked the app and did accessibility testing, everything is working fine.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Also, please merge your branch with latest develop code, as there was one issue which has been fixed right now.

File: app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/topic/practice/TopicPracticeFragment.kt]: Comment is updated.

File: app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt
Comment (Luffy18346) [reply app/src/main/java/org/oppia/app/topic/questionplayer/QuestionPlayerActivity.kt]: This comment is also updated.

**Comment by rt4914** [comment]
Screenshot 2020-01-23 at 3 13 54 PM

Your branch is still not up-to-date with develop. Please merge latest code from develop

File: app/src/main/res/layout/topic_practice_skill_view.xml
Comment (rt4914) [reply app/src/main/res/layout/topic_practice_skill_view.xml:20]: Undo this change, we need these lines of code.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Check these two comments:

https://github.com/oppia/oppia-android/pull/621#issuecomment-577604153

https://github.com/oppia/oppia-android/pull/621#discussion_r370017756

File: app/src/main/res/layout/topic_practice_skill_view.xml
Comment (rt4914) [reply app/src/main/res/layout/topic_practice_skill_view.xml:20]: Please reply to each and every comment/conversation.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Comment by rt4914** [comment]
From now on, as you are a collaborator, once a PR is approved, you will need to merge it own your own.

**Comment by Luffy18346** [comment]
@rt4914 Merging is blocked for me 
The message says "you are not authorized to merge this pull request".

**Comment by rt4914** [comment]
> @rt4914 Merging is blocked for me

Can you see `Squash and merge` button?

**Comment by Luffy18346** [comment]
> Can you see `Squash and merge` button?

no I cannot see the button. My merge button is disabled.

**Comment by rt4914** [comment]
> > Can you see `Squash and merge` button?
> > no I cannot see

Okay. No issues. btw, I have assigned you a new issue

**Pull request merged**
